envi/memcanvas/__init__.py

```diff
@@ -317,7 +317,7 @@ def renderMemoryAppend(self, size, cb=None):
 
         self._endRenderAppend(cb)
 
-    def _canvasCleared(self, cb, data):
+    def _canvasCleared(self, cb, sel, data):
```

Comment:
are you concerned that we're modifying the (albeit internal) API with an argument in the middle instead of at the end?  is there a reason?  it's not obvious to me the logic, but that doesn't mean there isn't any.